docker/Dockerfile.experimental

```diff
@@ -0,0 +1,12 @@
+# Stage 0, based on Node.js, to build and compile Angular
+FROM node:14.14.0 as node
+WORKDIR /app
+COPY package.json /app/
+RUN npm install
+COPY ./ /app/
+RUN npm run experimental:prod
```

Comment:
is this the only difference with default file?

Reply:
For now yes